Pipfile.lock

```diff
@@ -205,10 +220,10 @@
         },
         "defusedxml": {
             "hashes": [
-                "sha256:24d7f2f94f7f3cb6061acb215685e5125fbcdc40a857eff9de22518820b0a4f4",
-                "sha256:702a91ade2968a82beb0db1e0766a6a273f33d4616a6ce8cde475d8e09853b20"
+                "sha256:06d4515a8f8965624d6db922093eb11e77fb8f9a9ebedd1c5d6df5a0fcd0a12c",
+                "sha256:6c0b1461695877ececd6921a6a330e4392790275c5d6e88fc8ea8261445468b1"
             ],
-            "version": "==0.5.0"
+            "version": "==0.6.0rc1"
```

Comment:
Is rc version of defusedxml required for updating pydantic?

Reply:
updating lock did that, I think it's not for pydantic but for jupyter
```
 pipenv graph --reverse
defusedxml==0.6.0rc1
  - nbconvert==5.4.1 [requires: defusedxml]
    - jupyter==1.0.0 [requires: nbconvert]
    - notebook==5.7.8 [requires: nbconvert]
      - jupyter==1.0.0 [requires: notebook]
      - widgetsnbextension==3.4.2 [requires: notebook>=4.4.1]
        - ipywidgets==7.4.2 [requires: widgetsnbextension~=3.4.0]
          - jupyter==1.0.0 [requires: ipywidgets]
```

Reply:
So, guys, actually `Pipfile` and `Pipfile.lock` are irrelevant for the built package. They are there only for local development of FastAPI itself.

The only authoritative place that says what are the dependencies is the `pyproject.toml`.

Updating it in the `Pipfile` is helpful so that the local development environment reflects what is being defined in the official dependencies. But otherwise, it doesn't affect the final package.